buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -70,7 +73,11 @@ public void apply(Project project) {
               });
     }
     if (System.getenv().containsKey("METALAVA_BINARY_PATH")) {
```

Comment:
Seems like you no longer need this ENV check, right?

Reply:
Done